packages/p2p/src/pages/my-ads/my-ads-row-renderer.jsx

```diff
@@ -54,6 +58,7 @@ const MyAdsRowDropdown = ({
                     value: 'delete',
                 },
             ]}
+            list_portal_id={is_desktop ? `my_ads_table_row_${id}` : undefined}
```

Comment:
do we need to use the store value is_desktop here? we use the id only for desktop right? so can we not use id as an optional param and check if id exists, then assign it to my-ads-table-row-id else undefined.. can we try that?

Reply:
ah yeah true true, ill check this